container/docker/Dockerfile.base_image

```diff
@@ -38,13 +38,9 @@ RUN     apt-get update && \
 RUN     if [ "x${CONFIG_OPENVINO}" = "xy" ]; then \
 		apt-get update && \
 		apt-get -y install --no-install-recommends \
-		libnuma1 ocl-icd-libopencl1 && \
```

Comment:
not sure which apps use libnuma1? is it used by mysql? If yes, I don't think this should be removed.

Reply:
It's no need to control numa policy in CCAI docker container and it's impossible without pass privileges to container.
And also, client computer has only one CPU socket, it doesn't make sense to configurate numa policy because all memory are local.

Reply:
maybe this is a dependency of Openvino

Reply:
Openvino is NUMA awared, but does not depend on libnuma. libnuma is used by numactl (or other program) to control NUMA policy on multi-socket machine. I had examed openvino source code and libraries, they have no such dependency. I also tested fcgi and grpc apps.